modules/core/02-client/types/msgs_test.go

```diff
@@ -686,7 +685,7 @@ func (suite *TypesTestSuite) TestMsgRecoverClientValidateBasic() {
 }
 
 // TestMsgRecoverClientGetSigners tests GetSigners for MsgRecoverClient
-func TestMsgRecoverClientGetSigners(t *testing.T) {
+func (suite *TypesTestSuite) TestMsgRecoverClientGetSigners() {
```

Comment:
ditto to refactor test func, remove `suite` receiver and use `moduletestutil.MakeTestEncodingConfig`

same goes for each test in this file!